subprojects/wrapper-shared/src/test/groovy/org/gradle/wrapper/InstallTest.groovy

```diff
@@ -147,6 +171,42 @@ class InstallTest extends Specification {
         0 * download._
     }
 
+    def "get distribution hash"() {
```

Comment:
I'm not sure what's the convention for this, but I prefer method names which state what is being tested, so not "get distribution hash", but "distribution hash is fetched correctly", although this is not much better either... I'm not sure what would be best here, but for some reason I can't put my finger on I don't like these tests method names.